### DISCUSSION TITLE
securityonion-2.4.70-20240529.iso Failed to Open in VBox

### DISCUSSION THREAD

**Discussion by CoreyHebert-IT** [Security-Onion-Solutions/securityonion#13138] [2.4] [answered]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

2

### RAM

4

### Storage for /

50g total

### Storage for /nsm

50g total

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am working on obtaining my CySA+ and thought Security Onion would be good to add to my lab. I downloaded the .iso online, and attempted to add it to VBox, but the .iso filed is unable to read by Vbox. I'm running windows 11, 2 cores, 4g ram, 50g of space (I won't be running pcap capture for more than 20 minutes at a time.) an anyone point me in the right direction. 
![Screenshot 2024-06-04 174222](https://github.com/Security-Onion-Solutions/securityonion/assets/49260510/c0dcee40-426d-47a3-9832-8b908b7bd6f5)

Error: 

<html><body><p>Failed to open the disk image file <nobr><b>D:\Cyber\ISO\securityonion-2.4.70-20240529.iso</b></nobr>.</p><qt><p>Could not get the storage format of the medium <b><nobr>'D:\Cyber\ISO\securityonion-2.4.70-20240529.iso'</nobr></b> (VERR_NOT_SUPPORTED).</p>
Result Code: | VBOX_E_IPRT_ERROR (0X80BB0005)
-- | --
Component: | MediumWrap
Interface: | IMedium {ad47ad09-787b-44ab-b343-a082a3f2dfb1}
Callee: | IVirtualBox {7682d5eb-f00e-44f1-8ca2-99d08b1cd607}
Callee RC: | VBOX_E_OBJECT_NOT_FOUND (0X80BB0001)

<br></body></html>

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by err0r20** [comment, marked as answer]
rival,

Ensure that you check and verify the hash of the ISO. The ISO may not have downloaded correctly and/or could be corrupted. I would verify it downloaded correctly.
